### PR TITLE
feat: add ivan page

### PR DESCRIPTION
**Pull request by Donivanes**
This solves this issue #1 

- This PR add a new routing page to `/ivan`

### CODE REVIEW

**Review by Hector711 — APPROVED** [review]
OKEY

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -7,6 +7,9 @@ export const NavBar = () => {
         <li>
           <Link to="/">Go to home page</Link>
         </li>
+        <li>
+          <Link to="/ivan">Go to ivan page</Link>
+        </li>
```
Comment (Donivanes) [inline src/components/NavBar.jsx:12]: que haces tio

**Comment by MarcosUgalde** [comment]
I don't like your changes

**Review by MarcosUgalde — APPROVED** [review]
No me gusta tu código >:(

**Pull request merged**